packages/@react-spectrum/dialog/docs/Dialog.mdx

```diff
@@ -43,52 +47,345 @@ import {Heading, Text} from '@react-spectrum/typography';
+a ButtonGroup. While the Dialog will hide any elements occupying the footer section, it will not remove them from the DOM, 
+resulting in possible tab order issues.
+
+```tsx example
+<DialogTrigger isDismissable>
+  <ActionButton>Status</ActionButton> 
+  <Dialog>
+    <Header><Heading>Status</Heading></Header>
+    <Divider />
+    <Content>Printer Status: Connected</Content>
+  </Dialog>
+</DialogTrigger>
+```
+
+It is important to note that the `Heading`, `Content`, and `Footer` content elements accept any renderable 
+node, not just strings. This allows you to create Dialogs for more complex workflows, such as including a form
+for the user to fill out or adding confirmation checkboxes.
+
+```tsx example
+<DialogTrigger>
+  <ActionButton>Register</ActionButton> 
+  {(close) => (
+    <Dialog>
+      <Header><Heading>Register for newsletter</Heading></Header>
+      <Divider />
+      <Content>
+        <Form>
+          <TextField label="First Name" placeholder="John" autoFocus />
+          <TextField label="Last Name" placeholder="Smith" />
+          <TextField label="Street Address" placeholder="123 Any Street" />
+          <TextField label="City" placeholder="San Francisco" />
+        </Form>
+      </Content>
+      <Footer>
+        <Checkbox>
+          I want to receive updates for exclusive offers in my area.
+        </Checkbox>
+      </Footer> 
+      <ButtonGroup>
+        <Button variant="secondary" onPress={close}>Cancel</Button>
+        <Button variant="cta" onPress={close}>Register</Button>
+      </ButtonGroup>
+    </Dialog>
+  )}
+</DialogTrigger>
+```
 
 ### Accessibility
 
-*If the component has accessibility needs, include this section.*
+The tab order within the Dialog follows the order of the children provided to the Dialog. 
+This can be leveraged to create a logical flow separate from the reading order on the page.
+You are also responsible for determining what component, if any, should be automatically 
+focused on Dialog render. 
+
+The example below sets initial focus to the Dialog's `Save` button and establishes 
+the following tab order: `Save` button -> `Cancel` button -> `Textfield` -> `Checkbox`.
+
+```tsx example
+<DialogTrigger>
+  <ActionButton>Set Profile</ActionButton>
+  {(close) => (
+    <Dialog>
+      <Header><Heading>Profile</Heading></Header>
+      <Divider />
+      <ButtonGroup>
+        <Button autoFocus variant="cta" onPress={close}>Save</Button>
+        <Button variant="secondary" onPress={close}>Cancel</Button>
+      </ButtonGroup>
+      <Content>
+        <Form>
+          <TextField label="Name" />
+          <Checkbox>Make private</Checkbox>
+        </Form>
+      </Content>    
+    </Dialog>
+  )}
+</DialogTrigger>
+```
+
+### Advanced Usage
 
-### Internationalization
+The grid layout of a Dialog is made up of the following elements and sections, mapped to the corresponding default slot names:
```

Comment:
I don't want to get too technical with the slots here. It's more of an implementation detail. We should just document what components people should use for each with some examples for common patterns. For slots that don't map directly to components (e.g. `hero`), we can show how to use the `slot` prop as part of that example. Otherwise just use the component directly.

Maybe download the XD file from [here](https://spectrum-contributions.corp.adobe.com/page/standard-dialog-beta) and export the image with the slots to an SVG and embed here instead of the table? Would be a much more visual way to explain it I think.

Reply:
Ok will try that out, gonna update my computer so I can actually open the new XD file

Reply:
Looks like the slots image isn't in that XD file, gonna ask in the spectrum-react room for it. In the meantime, is there any other way of getting the svg to show up in the docs other than copy pasting the entire `<svg ...>` into the markdown directly? I tried using `![alt text](images/test.svg)` but no dice